### ISSUE TITLE
java.lang.Exception

### ISSUE THREAD

**Issue by dammyammy** [pestphp/pest-intellij#161]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Plugin version   | 1.0.3
| Pest version     | x.y.z
| IntelliJ version | PS-211.7628.25
| OS               | Mac OS X

### Stacktrace
```
java.lang.Exception
	at com.intellij.psi.stubs.StubTreeLoaderImpl.diagnoseLengthMismatch(StubTreeLoaderImpl.java:168)
	at com.intellij.psi.stubs.StubTreeLoaderImpl.checkLengthMatch(StubTreeLoaderImpl.java:134)
	at com.intellij.psi.stubs.StubTreeLoaderImpl.readFromVFile(StubTreeLoaderImpl.java:101)
	at com.intellij.psi.stubs.StubTreeLoader.lambda$stubTreeAndIndexDoNotMatch$0(StubTreeLoader.java:62)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$computeInNonCancelableSection$3(CoreProgressManager.java:237)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeNonCancelableSection(CoreProgressManager.java:218)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeInNonCancelableSection(CoreProgressManager.java:235)
	at com.intellij.psi.stubs.StubTreeLoader.stubTreeAndIndexDoNotMatch(StubTreeLoader.java:60)
	at com.intellij.psi.impl.source.FileTrees.reconcilePsi(FileTrees.java:202)
	at com.intellij.psi.impl.source.FileTrees.withAst(FileTrees.java:150)
	at com.intellij.psi.impl.source.PsiFileImpl.loadTreeElement(PsiFileImpl.java:210)
	at com.intellij.psi.impl.source.PsiFileImpl.calcTreeElement(PsiFileImpl.java:730)
	at com.intellij.psi.impl.source.PsiFileImpl.getNode(PsiFileImpl.java:933)
	at com.intellij.psi.impl.source.PsiFileImpl.getFirstChild(PsiFileImpl.java:740)
	at com.pestphp.pest.PestUtilKt.isPestTestFile(PestUtil.kt:19)
	at com.pestphp.pest.PestIconProvider.getIcon(PestIconProvider.kt:17)
	at com.intellij.util.PsiIconUtil.getProvidersIcon(PsiIconUtil.java:17)
	at com.intellij.psi.impl.ElementBase.doComputeIconNow(ElementBase.java:88)
	at com.intellij.psi.impl.ElementBase.lambda$computeIconNow$3(ElementBase.java:84)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:132)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:121)
	at com.intellij.psi.impl.ElementBase.computeIconNow(ElementBase.java:84)
	at com.intellij.psi.impl.ElementBase.lambda$static$0(ElementBase.java:44)
	at com.intellij.ui.DeferredIconImpl.evaluate(DeferredIconImpl.java:299)
	at com.intellij.ui.DeferredIconImpl.lambda$scheduleEvaluation$1(DeferredIconImpl.java:185)
	at com.intellij.ui.IconDeferrerImpl.evaluateDeferred(IconDeferrerImpl.java:110)
	at com.intellij.ui.DeferredIconImpl.lambda$scheduleEvaluation$2(DeferredIconImpl.java:185)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1096)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:79)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtil
```

**Comment by olivernybroe** [comment]
Could you explain what you did to get this error? :) 

Would be nice to know how to reproduce it.

**Comment by olivernybroe** [comment]
Issue has been fixed and will be released as part of v1.1.0.
You can find the latest release under releases tab for download manually or wait until the version has been approved in JetBrains Marketplace.
https://github.com/pestphp/pest-intellij/releases

Install it manually using
  <kbd>Preferences</kbd> > <kbd>Plugins</kbd> > <kbd>⚙️</kbd> > <kbd>Install plugin from disk...</kbd>
